apps/ecommerce/frontend/src/components/ResourceCard.tsx

```diff
@@ -79,10 +80,7 @@ const ResourceCard = (props: ResourceCardProps) => {
     );
   }
 
-  const provider = props.value.sys?.provider || 'External Resource';
-
-  let resourceType = props.value.sys?.linkType?.toString();
-  if (resourceType) resourceType = resourceType?.split(':')[1];
+  const [resourceProvider, resourceType] = props.value.sys?.linkType?.split(':');
```

Comment:
Seeing this makes me realize we'll need to eventually add more rigid runtime checking to this value, e.g. if someone uses an extra `:` in the provider or type, which would pass the type check but could produce unexpected results at run time.